libs/blocks/article-header/article-header.js

```diff
@@ -0,0 +1,184 @@
+import { createTag, getMetadata, getConfig } from '../../utils/utils.js';
+import { loadTaxonomy, getLinkForTopic, getTaxonomyModule } from '../article-feed/article-helpers.js';
+import { replaceKey } from '../../features/placeholders.js';
+import { fetchIcons } from '../../features/icons.js';
+import { buildFigure } from '../figure/figure.js';
+
+async function populateAuthorInfo(authorEl, imgContainer, url, name) {
+  if (!url) return;
+  const resp = await fetch(`${url}.plain.html`);
+  if (!resp || !resp.ok) {
+    const p = createTag('p', null, authorEl.textContent);
+    authorEl.replaceChildren(p);
+    console.log(`Could not retrieve metadata for ${url}`);
+    return;
+  }
+  const html = await resp.text();
+  const parser = new DOMParser();
+  const doc = parser.parseFromString(html, 'text/html');
+
+  const img = doc.querySelector('img');
+  if (img) {
+    img.setAttribute('alt', name);
+    imgContainer.append(img);
+    if (!img.complete) {
+      img.addEventListener('load', () => {
+        imgContainer.style.backgroundImage = 'none';
+      });
+      img.addEventListener('error', () => {
+        img.remove();
+      });
+    } else {
+      imgContainer.style.backgroundImage = 'none';
+    }
+  }
+}
+
+function openPopup(e) {
+  const target = e.target.closest('a');
+  const href = target.getAttribute('data-href');
+  const type = target.getAttribute('data-type');
+  window.open(
+    href,
+    type,
+    'popup,top=233,left=233,width=700,height=467',
+  );
+}
+
+async function copyToClipboard(button) {
+  try {
+    await navigator.clipboard.writeText(window.location.href);
+    button.setAttribute('title', await replaceKey('copied-to-clipboard', getConfig()));
+    button.setAttribute('alt', await replaceKey('copied-to-clipboard', getConfig()));
+    button.setAttribute('aria-label', await replaceKey('copied-to-clipboard', getConfig()));
+
+    const tooltip = createTag('div', { role: 'status', 'aria-live': 'polite', class: 'copied-to-clipboard' }, `${await replaceKey('copied-to-clipboard', getConfig())}`);
```

Comment:
Seems like this should be pulled out into a variable  
`const copiedToClipboard = await replaceKey('copied-to-clipboard', getConfig())`